crates/bevy_render/src/texture/mod.rs

```diff
@@ -12,15 +9,15 @@ pub use bevy_image::ExrTextureLoader;
 #[cfg(feature = "hdr")]
 pub use bevy_image::HdrTextureLoader;
 pub use bevy_image::{
-    BevyDefault, CompressedImageFormats, Image, ImageAddressMode, ImageFilterMode, ImageFormat,
-    ImageSampler, ImageSamplerDescriptor, ImageType, IntoDynamicImageError, TextureError,
-    TextureFormatPixelInfo,
+    BevyDefault, CompressedImageFormats, FileTextureError, Image, ImageAddressMode,
+    ImageFilterMode, ImageFormat, ImageFormatSetting, ImageLoader, ImageLoaderError,
+    ImageLoaderSettings, ImageSampler, ImageSamplerDescriptor, ImageType, IntoDynamicImageError,
+    TextureError, TextureFormatPixelInfo,
 };
 #[cfg(feature = "basis-universal")]
```

Comment:
it might make sense to to change the bevy_image import to a ::* to remove the basis-universal feature from bevy_render

Reply:
We still need the `basis-universal` feature to know whether we add the `CompressedImageSaver` to `bevy_render::ImagePlugin`, so I think we should hold off (to when/if we move the plugin to `bevy_image::ImagePlugin`)

I feel like we should just remove all these imports from `bevy_render` and have people depend on `bevy_image` directly, but that's another PR (and also might be more controversial since it bloats the migration guide).

Reply:
I also explicitly kept all of the "texture" formats as feature flags on `bevy_render` when I added them all to `bevy_image`, under the assumption that it's okay to have feature gates for specifically texture formats since they might require custom render options, whereas stuff like PNG will always be rendered just as pixels and thus doesn't need its own render flag.

So, it feels like `basis-universal` will always have a render flag because of that.